metatron-app/metatron-discovery
# #2584 disable spark engine when not properly configured

### Description
if dataprep.etl.spark option is disabled, 
the spark option should be unselectable on creating snapshot popup

**Related Issue** : 
[#2584 ](https://github.com/metatron-app/metatron-discovery/issues/2584)

### How Has This Been Tested?
1. Do not give spark option (refer below):
![스크린샷 2019-09-09 오후 2 52 22](https://user-images.githubusercontent.com/42257727/64506272-ec11b900-d311-11e9-8770-4ee15de86e82.png)
Just delete spark: option and its children.
2. Import & wrangle s5k_1.csv.txt
3. Click "Snapshot" button
4. Click "Advanced settings"
5. Spark should be disabled.


#### Need additional checks?


### Types of changes



### Checklist:


### Additional Context
